src/routes/(site)/genuary/collage.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+  import Dialog from './dialog.svelte';
+  import Media from './media.svelte';
+
+  import type { CollagePiece } from './collage';
+
+  export let data: CollagePiece[] = [];
+</script>
+
+<div class="grid-outer-wrapper">
+  <div class="grid-inner-wrapper">
+    {#each data as { src, view, alt: givenAlt }, i}
+      {@const alt = givenAlt || src}
+      {@const video = src.endsWith('.mp4') || src.endsWith('.mov')}
+      {@const ext = src.split('.').pop()}
+      <div class="artwork-container">
+        <div class="artwork">
+          <Dialog>
```

Comment:
The `Dialog` component requires JS. When JS fails to load, the user should be navigated to the full artwork in a new tab upon clicking.

Reply:
Partially resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/761/commits/3a79111341e214bdaa559ba5bc89baa31da762ee>.